examples/example_linbasex.py

```diff
@@ -15,13 +15,15 @@
 
 # Load image as a numpy array - numpy handles .gz, .bz2 
 IM = np.loadtxt("data/O2-ANU1024.txt.bz2")[::2,::2]
```

Comment:
Ah, sorry to have lowered your resolution :).  I'm sure you all worked hard to get the instrument up working so well, and it kills you to see it downsampled. I have a possible solutions:

In `setup.py`, we automatically detect if we are running on rtd. You could employ a similar strategy here, only downsampling if we are on rtd. I think you could try something like:
```python
import os
IM = np.loadtxt("data/O2-ANU1024.txt.bz2")
if os.environ.get('READTHEDOCS', None) == 'True':
    IM = IM[::2, ::2] # downsample image to avoid memory limitation of readthedocs.org
...
```